docs/02-app/01-building-your-application/03-rendering/04-composition-patterns.mdx

```diff
@@ -442,6 +442,7 @@ export default function ClientComponent({
     <>
       <button onClick={() => setCount(count + 1)}>{count}</button>
 
+      {/* You cannot import a Server Component into a Client Component. */}
```

Comment:
It's already highlighted on Line 432 when you import. It should error at the import not when used.

Reply:
I am aware of the existing highlighting. My feedback is that I found it non-obvious when scanning the docs. My suggestion in this PR is to make it more obvious. 

I wonder, what are the arguments against the first change (adding the comment to the highlighting)? And what for the second change (repeat the comment and highlight)?

Reply:
I wasn't aware that these are two separate changes. Highlighting the existing comment is fine. But moving the comment to a position where you won't get the error, is likely more confusing.

Reply:
@eps1lon how about this solution?
I really think having some comment at the the `<ServerComponent />` will help…